en/setup/dev_env_linux_arch.md

```diff
@@ -1,27 +1,48 @@
-# Development Environment on ArchLinux
+# Development Environment on Arch Linux
 
-> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets, using an unsupported version of GCCE from the package manager.
-The instructions have been tested on Antergos (an Arch Linux based distribution) as it is easier to set up than Arch Linux.
-We hope to provide fully tested instructions with the supported toolchain in the near future.
+> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim or gazebo.
+The instructions have been tested on Manjaro (an Arch Linux based distribution) as it is much easier to set up than Arch Linux.
 
-## Permissions
+## Setup script
 
-The user needs to be added to the group "uucp":
+Using the the provided setup script it's very convenient to set your Arch installation up for PX4 development. Here are the steps to get you started from scratch.
 
+### Run the setup script
+The script by default installs all tools to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim.
+
+Optionally you can specify the parameter `--gazebo` to also install gazebo simulation from the [AUR](https://aur.archlinux.org/packages/gazebo/). Note that gazebo gets compiled from source and therefore takes some time to install and requires entering the sudo password multiple times for dependencies.
+
+#### Option 1 Clone PX4, install Toolchain
 ```sh
-sudo usermod -a -G uucp $USER
+git clone --recurse-submodules -j8 https://github.com/PX4/Firmware.git
```

Comment:
Submodules are cloned on demand during compilation, aren't they? 
`--recursive-submodules` is useful but takes time and disk space for someone who just wants to install the toolchain.

Reply:
Are they reliably? I always clone all of them. 😇 TIL 👍

Reply:
Yes it works reliably. My typical workflow (often jumping between various old branches and back to master) is to `make distclean` and let the next build reinit the subset of submodules actually used by that particular config.

Reply:
Nice! Then we could add gtest as a submodule as well. I was only doing the configure time hackery because I didn't want everyone to clone it even though they're not testing.

Reply:
Fixed that.